docs/guide/custom_policies.md

```diff
@@ -118,6 +118,32 @@ class AcmeExampleBundle extends Bundle
 }
 ```
 
+If you are using Symfony 5 you have to use the `App\Kernel` to register a new policy provider :
```

Comment:
as this is targeting 3.1 and higher, do we need to mention things like `if you are suing sf 5` ?

Reply:
Can replace Symfony 5, EzPlatform 3.1

Reply:
That is not my point, your suggestion towards 3.1 doc will result in 3.1 doc containing two instructions for how to do this. One which you seems to indicate does not work on 3.1 and then a second way for 3.1

Summary: 3.1 branch is 3.1 doc => 3.1 doc only need instructions for how you do things on 3.1, we don't need both here.